src/app/(home)/meeting/_components/AttendeeProfiles.tsx

```
@@ -1,35 +1,68 @@
-import { getAttendee } from '@/api/getAttendee';
+'use client';
+import { getMeetingAttendee } from '@/api/meeting/getMeetingAttendee';
 import { useEffect, useState } from 'react';
 import Image from 'next/image';
+import { IUser } from '@/types/userType';
+
+interface MeetingAttendee {
```

ESLint

[error] 'MeetingAttendee' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)